src/renderer/components/AnnouncementProviderWrapper.js

```diff
@@ -67,6 +68,8 @@ export function AnnouncementProviderWrapper({ children }: Props) {
     language,
     currencies,
     getDate: () => new Date(),
+    lastSeenDevice: lastSeenDevice || undefined,
+    platform: process.platform,
```

Comment:
process.platform is one of 'aix', 'darwin', 'freebsd', 'linux', 'openbsd', 'sunos', 'win32' ( https://nodejs.org/api/process.html#process_process_platform )

which is not what you seem to expect at
https://github.com/LedgerHQ/ledger-live-common/blob/84d9acd711476c0ab95294f29918ad22bc87468c/src/notifications/AnnouncementProvider/logic.js#L22